src/pages/location/createLocation/components/AddBinLocationDialog.ts

```diff
@@ -0,0 +1,13 @@
+import BasePageModel from '@/pages/BasePageModel';
+
+class AddBinLocationDialog extends BasePageModel {
+  get binLocationNameField() {
+    return this.page.locator('#dlgAddBinLocation').locator('#name');
+  }
```

Comment:
I would like to separate this to have sth like:
```
 get binLocationDialog() {
    return this.page.locator('#dlgAddBinLocation');
  }
```
and then use the binLocationDialog in the binLocationNameField

Reply:
AddBinLocationDialog is already a separate component and  binLocationNameField is its element. Why should I add binLocationDialog on AddBinLocationDialog if it is the same dialog?